app/src/main/java/com/spencerpages/MainActivity.java

```diff
@@ -130,19 +135,35 @@
 
     // Export directory path
     private final static String EXPORT_FOLDER_NAME = "/coin-collection-app-files";
-	
+
+    // Default list item view positions
+    // Note: Using constants instead of an enum based on this:
+    // https://developer.android.com/training/articles/memory.html#Overhead
+    // - Enums often require more than twice as much memory as static constants.
+    public final static int ADD_COLLECTION = 0;
+    public final static int REMOVE_COLLECTION = 1;
+    public final static int IMPORT_COLLECTIONS = 2;
+    public final static int EXPORT_COLLECTIONS = 3;
+    public final static int REORDER_COLLECTIONS = 4;
+    public final static int ABOUT = 5;
+
```

Comment:
Would you mind moving NUMBER_OF_COLLECTION_LIST_SPACERS to be defined near these?